taskiq/cli/worker/process_manager.py

```diff
@@ -159,7 +159,7 @@ def __init__(
         self,
         args: WorkerArgs,
         worker_function: Callable[[WorkerArgs], None],
-        observer: Observer | None = None,  # type: ignore[valid-type]
+        observer: Optional[Observer] = None,  # type: ignore[valid-type] # noqa: UP045
```

Comment:
```suggestion
        observer: "Observer | None" = None,  # type: ignore[valid-type]
```

We can change it to this. This way we don't need noqa.